Formula/azure-storage-cpp.rb

```diff
@@ -0,0 +1,23 @@
+class AzureStorageCpp < Formula
+  desc "Microsoft Azure Storage Client Library for C++"
+  homepage "https://azure.github.io/azure-storage-cpp"
+  url "https://github.com/Azure/azure-storage-cpp/archive/v5.1.1.tar.gz"
+  sha256 "29b4b0c47784ae541b0ef1b3693acba54e0d06bac78d139105a379ab1fd20333"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "cpprestsdk"
+  depends_on "gettext"
+  depends_on "libxml2"
```

Comment:
Have you tried without libxml2? (i.e. using the system libxml2)

Reply:
it's hard to say if it will work since my system is already provisioned with several formulae that need libxml2. The official [documentation](https://github.com/Azure/azure-storage-cpp#getting-started-on-osx)
 says to install it - so I followed their lead.

Is there a way to build the formula in something like a "chroot" environment to check this? Is it really necessary to get rid of this dependency?

Reply:
Try to push it without this line, and we will see on our CI machines whether it works.

Reply:
seems to work with system libxml2 as well. Removed dependency with revision 83c8cc495078c2557ae9c393d8795a0881dbc7cb